src/main/java/io/cdap/delta/bigquery/BigQueryAssessor.java

```diff
@@ -61,6 +65,20 @@ public TableAssessment assess(StandardizedTableDetail tableDetail) {
     List<Problem> problems = new ArrayList<>();
     String dbName = tableDetail.getDatabase();
     String tableName = tableDetail.getTable();
+
+
+    if (tables.containsKey(tableName)) {
+      // TODO support same table name in different schema
+      if (!Objects.equals(tables.get(tableName), tableDetail.getSchemaName())) {
+        problems.add(new Problem("Duplicate Table Name", String.format("Duplicated table name %s detected. BigQuery " +
+          "doesn't support duplicate table name even though they are in different schema", tableName),
+          "Please only select one of the tables with same table name to replicate",
+          "Not be able to replicate multiple tables with same name " + "to BigQuery"));
+      }
```

Comment:
should we return from here? Since this case is not supported, no need to check additional details such as primary keys for example.

Reply:
I think we'd better expose all the problems. Because we don't know which table the user finally will choose. For example, if user choose schema1.table1 and schema2.table1 , if we return here , we will report all the problem for schema1.table1 and only report duplicate table name problem for schema2.table2 which is not consistent. 

Reporting all the problem will have below benefits:
1) Give user more information to help them decide which table to choose finally
2) User correct the problem before they choose that table (the second table)  instead of choose the table and see new problems.

Reply:
we should expose problems. Think of the user, at least they will know the issue. If we can also suggest corrective actions that will provide an experience far better than just silently ignoring it or conveying problem that user has no clue of.

Reply:
+1 to the consistency point. Thanks for explanation Sean! Agree even this case is not supported its better to expose all the problems.